README.rst

```diff
@@ -259,20 +259,24 @@ to install using pip and a virtualenv::
     python3 -m venv ./env
     source ./env/bin/activate
     pip install -e ".[all,test]"
-
+    
 This will run a process of downloading and installing all the needed
 dependencies into a virtual env.
 
+Some of the dependencies might have to be installed manually, if you run into errors just install the dependency that is mentioned in the error manually.
```

Comment:
I'm curious - in what case was this necessary for you? What errors did you see?

Reply:
Someone answered this in #synapse-dev, stating that it helped to install dependencies individually when using `export HTTP_PROXY` on their system.

I'm aware this can also be an issue with resource-constrained devices - installing everyone at once can cause `pip` to overwhelm the system's available memory.

Reply:
I don't remember the exact modules that were necessary, but there were definitely a few...